notebooks/real-time/rtcloud_notebook.ipynb

```diff
@@ -299,17 +339,249 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "Instead of running the classification script from the web browser (as above), it can be run from the command line. The command below will run the sample.py script (of the real-time aperture notebook) as if running on the command line."
+    "Instead of running the classification script from the web browser (as above), it can be run directly by running the scripts main function. "
    ]
   },
   {
    "cell_type": "code",
    "execution_count": null,
    "metadata": {},
-   "outputs": [],
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "Location of the subject's dicoms: /tmp/notebook-simdata\n",
+      "\n",
+      "\n",
+      "\n",
+      "Clear any pre-existing plot for this run using 'clearRunPlot(runNum)'\n",
+      "###################################################################################\n",
+      "/tmp/notebook-simdata/labels.npy\n",
+      "Collected training data for TR 0\n",
```

Comment:
Hi David, Can you clear the "outputs" created from running the notebook before checking in? I think it's in the Jupiter menu Cell->clear->all_outputs.

Reply:
Done